.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
this file should not be added to the repo, you can add it to the global .gitignore file: `/home/username/.gitignore`